loom_js_test/src/challenge-after-demo.ts

```diff
@@ -86,15 +87,18 @@ export async function runChallengeAfterDemo(t: test.Test) {
 
   // Mallory's exit should be auto-challenged by Dan's client, but watching/auto-challenge hasn't
   // been implemented yet, so challenge the exit manually for now...
-  await dan.challengeAfterAsync({ slot: deposit1Slot, challengingBlockNum: plasmaBlock3 })
+  // console.log('DAN WOULD CHALLENGE WITH', plasmaBlock3)
+  // await dan.challengeAfterAsync({ slot: deposit1Slot, challengingBlockNum: plasmaBlock3 })
 
   // Having successufly challenged Mallory's exit Dan should be able to exit the coin
+  await sleep(2000)
   await dan.startExitAsync({
     slot: deposit1Slot,
     prevBlockNum: coin.depositBlockNum,
     exitBlockNum: plasmaBlock3
   })
-  //dan.stop_watching_exits(deposit1_utxo)
+
+  dan.stopWatching(deposit1Exit, console.log(`Stopped watching ${deposit1Slot}`))
```

Comment:
I've changed the signature for this function, so it returns a promise now, this should be rewritten like so:
```typescript
await dan.stopWatchingAsync(deposit1Exit)
console.log(`Stopped watching ${deposit1Slot}`)
```